core/executor/src/wasm_executor.rs

```diff
@@ -66,6 +66,14 @@ impl<'e, E: Externalities<Blake2Hasher>> FunctionExecutor<'e, E> {
 			hash_lookup: HashMap::new(),
 		})
 	}
+
```

Comment:
Here I did not implement a wasm api that could use a cache subtrie. This is required as currently if runtime is wasm we would query subtrie for every access.
I need confirmation of the right way to use an api with `SubTrie`: I started some memory allocated on native and refer to it by a void pointer but I guess it break security requirement, so probably better to have it copied on each function call (still the cost seems to big so there might be some middle ground).
Maybe also the fact that we fetch subtrie on every query is good design : that is if we do not trust the input from the call (I did not expect it to be the case for runtime).
At this point it seems to me that subtrie needs its own runtime wasm memory instantiation, but I keep that for a next issue to keep the PR small.

Reply:
resolved as an issue was created for it.